### DISCUSSION TITLE
Rust tools apt repository

### DISCUSSION THREAD

**Discussion by emmatyping** [BurntSushi/ripgrep#2072] [General] [answered]
Hi! I wanted to share I set up an APT repository for popular Rust tools, which of course includes ripgrep.

The details are at https://apt.cli.rs. I take the exact binary published on the Github release to use as a package.

Would a PR to update the README to mentioning this as a means of installing the latest ripgrep be of interest?

**Comment by BurntSushi** [comment, marked as answer]
I'm open to it. One concern is that it becomes unmaintained and folks following along in the README might not realize it. How long have you been maintaining it? And how long do you plan on maintaining it?

Another concern is how to sell it. Why would, for example, an Ubuntu or Debian user use this? Presumably they're on such a Linux distro because they _don't_ want the latest and greatest just scraped from the web, but from their friendly neighborhood package maintainer.

**Reply by emmatyping** [reply]
I've been maintaining it for a few months now, and I plan on maintaining it into the indefinite future. I have plans to automate it mostly or completely.

I agree Debian based distro users often want stability, but there are those like Pop!_OS (and even Ubuntu) that are used for desktop usage, where stability is not as important. One may also want to grab the latest ripgrep for a handy feature or bug fix that hasn't been backported.

**Reply by BurntSushi** [reply]
I'm still skeptical, because why choose Ubuntu if you just want to get the latest fixes and features?

In any case, feel free to submit a PR. My guess is that wording for this might be a little tricky, because it needs to be clear why or when someone would want this.

**Reply by emmatyping** [reply]
I personally use Ubuntu because the base OS is very stable, but with PPAs and apt repositories I can opt into newer versions of packages. Perhaps that is unusual :)
